### ISSUE TITLE
Segment overflow with three buttons only.

### ISSUE THREAD

**Issue by dev-pacific22** [GeekyAnts/NativeBase#1917]
Segments overflowing in parent view with only three buttons.
It looks like as:

![simulator screen shot - iphone 6 - 2018-05-22 at 12 23 01](https://user-images.githubusercontent.com/23580199/40346566-e3c63276-5dba-11e8-9140-baf69d3b83b7.png)

Can not apply the styles also.

**Comment by akhil-geekyants** [comment]
@dev-pacific22 try changing segment button's padding (paddingLeft and paddingRight).
Sample code
```
import React, { Component } from 'react';
import { StyleSheet } from "react-native";
import { Container, Header, Left, Body, Right, Button, Icon, Title, Segment, Content, Text } from 'native-base';
export default class App extends Component {

  state = { selected: 1 }

  render() {
    return (
      <Container>
        <Header>
          <Left>
            <Button transparent>
              <Icon name="arrow-back" />
            </Button>
          </Left>
          <Body>
            <Title>Forms</Title>
          </Body>
          <Right>
            <Button transparent>
              <Icon name="search" />
            </Button>
          </Right>
        </Header>
        <Segment>
          <Button first active={this.state.selected === 1} style={styles.segmentButton} onPress={() => this.setState({ selected: 1 })}>
            <Text >Get Started</Text>
          </Button>
          <Button active={this.state.selected === 2} style={styles.segmentButton} onPress={() => this.setState({ selected: 2 })}>
            <Text>In Progress</Text>
          </Button>
          <Button last active={this.state.selected === 3} style={styles.segmentButton} onPress={() => this.setState({ selected: 3 })}>
            <Text>Completed</Text>
          </Button>
        </Segment>
        <Content padder>
          <Text>Segment</Text>
        </Content>
      </Container>
    );
  }
}

const styles = StyleSheet.create({
  segmentButton: {
    paddingLeft: 0,
    paddingRight: 0
  }
})
```
![simulator screen shot - iphone 6 - 2018-05-23 at 17 10 32](https://user-images.githubusercontent.com/27768184/40422299-93586f6e-5eac-11e8-87d2-28fc77b21470.png)

**Comment by SupriyaKalghatgi** [comment]
Released with [2.5.0](https://github.com/GeekyAnts/NativeBase/tree/v2.5.0)
